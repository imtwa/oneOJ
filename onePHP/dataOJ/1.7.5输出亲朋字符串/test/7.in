5%2/")=&98**%%:'1&9(09,'8"!*',&>72$8 5! >53,51;=3='3$'5>.3/3"(.+9/6 =#&<)38:8=<?/,