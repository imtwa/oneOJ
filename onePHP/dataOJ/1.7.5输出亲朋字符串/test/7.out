ZWaQKfc_qbTOJ_aXW_aXieS_ZA@KQSRduiVBVXUVA?]shQJaflxppdZWK\slabbUJF<LYdheV]`Ibe\krruy{n[a